### PR TITLE
Fix simple_rebuild_test

### PR DESCRIPTION
**Pull request by yukim**
Current test implementation didn't trigger rebuild concurrently even though we are using thread in my local laptop. I think the same thing is happening in CI since it is hitting assertion error that indicates it.

So I changed the code to run nodetool in background first, and then immediately trigger second one in foreground, and the latter should fail at that point.

Rebuild completion from the former is guarded by wathing log.

### CODE REVIEW

File: rebuild_test.py [hunk 1]
```diff
@@ -78,55 +77,21 @@ def simple_rebuild_test(self):
...
         # exactly 1 of the two nodetool calls should fail
-        # usually it will be the one in the main thread,
-        # but occasionally it wins the race with the one in the secondary thread,
-        # so we check that one succeeded and the other failed
-        self.assertEqual(self.rebuild_errors, 1,
-                         msg='rebuild errors should be 1, but found {}. Concurrent rebuild should not be allowed, but one rebuild command should have succeeded.'.format(self.rebuild_errors))
+        try:
+            node2.nodetool('rebuild dc1')
+        except NodetoolError as e:
+            self.assertTrue('Node is still rebuilding' in e.message)
```
Comment (mambocab) [inline rebuild_test.py]: This should fail if the `nodetool` call doesn't raise an error.

Comment (yukim) [inline rebuild_test.py]: You are right, pushed fix.

**Comment by pauloricardomg** [comment]
+1

**Comment by mambocab** [comment]
This is great, sorry to miss the final +1. We in TE should have seen this. Thank you both!

**Pull request merged**